### PR TITLE
Update loadbalancer with AWS clarification

### PR DESCRIPTION
**Pull request by jsenon**
Add special notes for AWS ELB thanks to @Johanan Liebermann and @Hubert Chan

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.9 Features: set Milestone to `1.9` and Base Branch to `release-1.9`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.
>
> NOTE: After opening the PR, please *un-check and re-check* the "Allow edits from maintainers" box. This is a temporary workaround to address a known issue with GitHub.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6225)

### CODE REVIEW

**Comment by jsenon** [comment]
CLA Signed

File: docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -462,6 +462,8 @@ Special notes for Azure: To use user-specified public type `loadBalancerIP`, a s
 public IP address resource needs to be created first, and it should be in the same resource
 group of the cluster. Then you could specify the assigned IP address as `loadBalancerIP`.
 
+Special notes for AWS: AWS ELB only know instance, it doesn't know POD. When you use k8s service with `type: LoadBalancer`, kube-proxy will listen on a randomly port on instance in the cluster, an ELB is created and mapped to each instances ports even if a pod, which serve service, is not present. ELB uses round robin for the loabalancing mecanism. kube-proxy (on behalf of srevice) use a random connection
```
Comment (tengqm) [inline docs/concepts/services-networking/service.md]: suggestions:

- wrap your lines at about 80 characters boundary so the PR is easy to review/comment
- check the `Allow edits from maintainers` button so some native speaker reviewers can do a quick edit on your PR
- be careful about some obvious typos before pushing your PR

Comment (jsenon) [reply docs/concepts/services-networking/service.md]: Thanks @tengqm I've take into account your remarks for future PR

File: docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -462,6 +462,8 @@ Special notes for Azure: To use user-specified public type `loadBalancerIP`, a s
 public IP address resource needs to be created first, and it should be in the same resource
 group of the cluster. Then you could specify the assigned IP address as `loadBalancerIP`.
 
+Special notes for AWS: AWS ELB only know instance, it doesn't know POD. When you use kubernetes service with `type: LoadBalancer`, kube-proxy will listen on a randomly port on instance in the cluster.  An ELB is created and mapped to each instances ports even if a pod, which serve service, is not present. ELB uses round robin for the loabalancing mecanism. kube-proxy (on behalf of service) use a random connection.
```
Comment (tengqm) [inline docs/concepts/services-networking/service.md]: listen on a randomly port on instance -> listen on a random port on an instance

**Comment by tengqm** [comment]
still, the new paragraph reads strange ...
please rebase.

**Comment by zhangxiaoyu-zidif** [comment]
@jsenon 
Mr. @tengqm want you polish this PR, and make the sentence read smooth. =)
Thanks for your contributions.

**Comment by heckj** [comment]
I'm closing this PR since there's been no activity for months

**Pull request closed without merging**